src/SignalR/common/Shared/ReusableUtf8JsonWriter.cs

```diff
@@ -20,7 +21,7 @@ internal sealed class ReusableUtf8JsonWriter
 
         public ReusableUtf8JsonWriter(IBufferWriter<byte> stream)
         {
-            _writer = new Utf8JsonWriter(stream, new JsonWriterOptions() { SkipValidation = true });
+            _writer = new Utf8JsonWriter(stream, new JsonWriterOptions() { SkipValidation = true, Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping });
```

Comment:
nit: Consider turning Validation **on** in DEBUG (or put another way, only turn it off in RELEASE), similar to how the serializer itself does it:
https://github.com/dotnet/corefx/blob/bc10b53838bc84ac2b626e16eee1338183ee55c8/src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs#L348-L350